docs/developer/devicesimulator.md

```diff
@@ -27,14 +260,16 @@ To simulate a large number of devices (10,000 for example) a
 
 Run the same commands per each host. Note: some commands must be run remotely.
 
+```sh
     # on remote host
     mkdir -p ~/.flightctl/certs/ ~/.config/flightctl ~/bin
     scp bin/agent/etc/flightctl/certs/* your_username@remotehost:~/.flightctl/certs/
     scp bin/agent/etc/flightctl/config.yaml your_username@remotehost:~/.flightctl/agent.yaml
     scp ~/.config/flightctl/client.yaml your_username@remotehost:~/.config/flightctl/client.yaml
     scp bin/devicesimulator your_username@remotehost:~/bin/
     # on remote host 
-    chmod -R 755 ~/.flightctl/ ~/.config/flightctl bin/devicesimulator
+    chmod -R 755 ~/.flightctl/ ~/.config/flightctl ~/bin/devicesimulator
```

Comment:
Are you sure about this change ?

Reply:
It probably isn't necessary, but based on the setup in the previous steps everything should be available in the home directory.